Apps/Src/Voltage.c

```diff
@@ -130,12 +129,13 @@ void Voltage_OpenWireSummary(void){
  */
 SafetyStatus Voltage_OpenWire(void){
 	wakeup_idle(NUM_MINIONS);
-	long openwires = LTC6811_run_openwire_multi(NUM_MINIONS, Minions, false);
-	if(openwires != 0){
-		return DANGER;
-	} else {
-		return SAFE;
+	LTC6811_run_openwire_multi(NUM_MINIONS, Minions, false);
+
+	for(int32_t i = 0; i < NUM_MINIONS; i++) {
+
```

Comment:
Should there be something in this for loop to read the updated Minions struct? Won't this always return danger otherwise?

Reply:
I placed the same comment above: https://github.com/lhr-solar/BPS/pull/210#discussion_r495595308